src/Microsoft.ML.StandardLearners/Standard/MultiClass/doc.xml

```diff
@@ -47,10 +43,6 @@
         request caching, as it will be performing multiple passes over the data set.
         These learner will request normalization from the data pipeline if the classifier indicates it would benefit from it.</para>
       </remarks>
-      <seealso cref='Microsoft.ML.Legacy.Trainers.LogisticRegressionClassifier'/>
-      <seealso cref='Microsoft.ML.Legacy.Trainers.LightGbmClassifier'/>
-      <seealso cref='Microsoft.ML.Legacy.Trainers.StochasticDualCoordinateAscentClassifier'/>
-      <seealso cref='Microsoft.ML.Legacy.Trainers.NaiveBayesClassifier'/>
```

Comment:
Same here.

Reply:
@sfilipi had asked to swap these names out in the previous PR.